chains/avalanche/AvalancheLib.sol

```diff
@@ -195,17 +195,7 @@ library AvalancheLib {
     }
 
     function _addStrategyLogic(IFactory factory, string memory id, address implementation, bool farming) internal {
```

Comment:
remove this internal method, use factory.setStrategyImplementation directly in code